docs/flow/config-language/syntax.md

```diff
@@ -6,3 +6,88 @@ weight: 200
 ---
 
 # Syntax
```

Comment:
Somewhere in here I think we need to explain "terminators," since terminators are needed after every attribute and block. Terminators will show up in error messages for the lexer/parser, so I think we need to define it (I thought about changing the error message but I'm not confident) 

Terminators are currently defined as newlines. Terminators are found if there is a newline following an expression, `]`, `)`, or `}`. Other newlines are effectively ignored and a user may enter as many newlines as they want.